action.yml

```diff
@@ -18,53 +21,62 @@ inputs:
     description: 'Matrices nested levels count (from 1 to 3)'
```

Comment:
Is this how many nested levels of matrices are returned as an output of the action?

Reply:
yes

Reply:
Please update description to say:

"Number of nested matrices that should be returned as the output"

Reply:
Done